src/ess/reduce/nexus/generic_types.py

```diff
@@ -151,8 +152,11 @@ class MonitorData(
 Component = TypeVar('Component', bound=snx.NXobject)
 
 
+class Filename(sciline.Scope[RunType, Path], Path): ...
```

Comment:
With this, there are now to `Filename` types. The other one is in `types.py`. This seems confusing.

Reply:
Please see https://github.com/scipp/essreduce/pull/75#discussion_r1719588569, this is the case for about 20 or so types.

Reply:
See me new comment